packages/trader/src/Modules/Contract/Components/AccumulatorsStats/accumulators-stats.tsx

```diff
@@ -55,7 +62,14 @@ const AccumulatorsStats = ({ is_dark_theme, is_expandable = true, ticks_history_
                     {!is_collapsed ? (
                         <div className='accumulators-stats__history-heading'>{localize('Number of ticks')}</div>
                     ) : (
-                        rows[0]?.map((el, i) => <TicksHistoryCounter key={i} value={el} has_progress_dots={i === 0} />)
+                        rows[0]?.map((el, i) => (
+                            <TicksHistoryCounter
```

Comment:
`TicksHistoryCounter` (packages/trader/src/Modules/Contract/Components/AccumulatorsStats/ticks-history-counter.jsx) has it's own `data-testid='dt_accu_stats_history_counter'`. Is it possible to use it instead of adding one more?